File: testing/testing/Classes/Controllers/ViewControllers/ViewController.m
//
//  ViewController.m
//  Testing
//
//  Created by dev on 24/1/17.
//  Copyright (c) 2017 dev. All rights reserved.
//

#import "AppConfig.h"
#import "ViewController.h"
#import "PListUsers.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    [self->lblText setText:@"some user"];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (IBAction)handleButtonClick:(id)sender {
    self->lblText.text = self->txtInput.text;
}

- (void)touchesBegan:(NSSet *)touches withEvent:(UIEvent *)event {
     [self.view endEditing:YES];
//    [self.txtInput resignFirstResponder];
}

-(void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
    [[PListUsers current] setUser: [[PListUsers current] getUserByName:self->lblText.text]];
}

@end

File: testingSwift/testingSwift/ViewController.swift
//
//  ViewController.swift
//  testingSwift
//
//  Created by dev on 23/2/17.
//  Copyright (c) 2017 dev. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet var lblText: UILabel!
    @IBOutlet var txtInput: UITextField!
    @IBOutlet var btnClick: UIButton!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        self.view.endEditing(true)
        self.lblText.text = "something"
    }
    
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    
    override func touchesBegan(_ touches: NSSet, with event: UIEvent?) {
        self.view.endEditing(true)
    }
    
    @IBAction func handleButtonClick( _ sender: AnyObject) {
        self.lblText.text = self.txtInput.text
    }
    
    /*
    // MARK: - Navigation

    // In a storyboard-based application, you will often want to do a little preparation before navigation
    override func prepareForSegue(segue: UIStoryboardSegue, sender: AnyObject?) {
        // Get the new view controller using segue.destinationViewController.
        // Pass the selected object to the new view controller.
    }
    */

}
